proto/spire/api/node/node.proto

```diff
@@ -135,6 +135,15 @@ message FetchX509CASVIDResponse {
     spire.common.Bundle bundle = 2;
 }
 
+message PushJWTKeyUpstreamRequest {
+    spire.common.PublicKey jwt_key = 1;          
+}
+
+message PushJWTKeyUpstreamResponse {
+    /** up-to-date bundle of JWT signing keys */
```

Comment:
nit: comment style doesn't match the other fields in this file
```suggestion
    // up-to-date bundle of JWT signing keys
```

Reply:
Oh damn, I guess I copy & pasted from the wrong example (common.proto). Suggestion committed.